challenge.js

```diff
@@ -42,7 +42,7 @@
  */
 
 // ANSWER
-
+A D E {}
```

Comment:
Very close - `body div p`. However, this assumes that there are no other layers of nesting in between. If we want to be extra precise, we might write `body > div > p`.